playwright.config.ts

```diff
@@ -6,7 +6,9 @@ import { defineConfig, devices } from '@playwright/test';
  */
 // require('dotenv').config();
 
-const baseURL = process.env.PLAYWRIGHT_TEST_BASE_URL || 'http://localhost:3000';
+const baseURL =
+  process.env.PLAYWRIGHT_TEST_BASE_URL ||
+  'https://pr-301---ms-server-staging-c4f6qdpj7q-ew.a.run.app';
```

Comment:
little oversight :)

Reply:
Woops. Fixed it in the next commit.
Should i keep retain on failure? It was useful to get the correct traces for the flaky tests

Reply:
Yeah, don't see why not.